Orange/widgets/data/oweditdomain.py

```diff
@@ -2420,11 +2430,8 @@ def abstract(var):
 
 def _parse_attributes(mapping):
     # type: (Iterable[Tuple[str, str]]) -> Dict[str, Any]
-    # Use the same functionality that parses attributes
-    # when reading text files
-    return Orange.data.Flags([
-        "{}={}".format(*item) for item in mapping
-    ]).attributes
+    # Map to dict to preserve booleans
+    return dict((key, value) for key, value in mapping)
```

Comment:
```Orange.data.Flags``` use here was added to match the .*tab* annotations row parsing. E.g. it parses number strings to integers, ..., but not True/False booleans. The tests fail because it no longer does this mapping due to this change.

I think the boolean mapping/parsing should be moved to the `Orange.data.Flags`. That way it will preserve the value type on save/load.

Reply:
Ok, thanks for the hint! Will have a look.